runner/debug/debug_test.go

```diff
@@ -0,0 +1,109 @@
+package debug
+
+import (
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/hcdiag/redact"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVaultCmdString(t *testing.T) {
+	tcs := []struct {
+		name            string
+		cfg             VaultDebugConfig
+		productDuration time.Duration
+		productInterval time.Duration
+		filterString    string
+		expected        string
+	}{
+		{
+			name: "product config defaults should be used when no configuration is passed in (duration and interval)",
+			cfg: VaultDebugConfig{
+				Compress:        "false",
+				LogFormat:       "standard",
+				MetricsInterval: "10s",
+				Targets:         []string{},
+				Redactions:      []*redact.Redact{},
+			},
+			productDuration: 5 * time.Minute,
+			productInterval: 45 * time.Second,
+			filterString:    "",
```

Comment:
I think these `filterString` fields can be removed when they have empty strings, since that's the default for a field of type `string`.